test/src/test/java/se/fortnox/reactivewizard/examples/junit5_examples/Junit5Tests.java

```diff
@@ -0,0 +1,35 @@
+package se.fortnox.reactivewizard.examples.junit5_examples;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.function.Supplier;
+
+class Junit5Tests {
```

Comment:
I think we should remove these pure example test cases. We can state in the reactive wizard documentation in backstage that junit5 is supported out of the box maybe.